tensorflow_recommenders_addons/dynamic_embedding/core/kernels/dynamic_partition_op_gpu.cu.cc

```diff
@@ -120,9 +120,9 @@ void MoveValues(const GPUDevice& d, int32* keys, int32* values, int32* num_runs,
 }
 
 struct IdentityOp {
-  __device__ int32 __forceinline__ operator()(const int32& a) const {
-    return a;
-  }
+  // __device__ int32 __forceinline__ operator()(const int32& a) const {
```

Comment:
Why are these lines commented out? Could you please add a note explaining the reason for commenting them, to aid in future maintenance?

Reply:
It looks like I'd be better off removing them. I didn't make it because the commented code is copied from the tensorflow source code.